package.json

```diff
@@ -1,23 +1,15 @@
 {
   "name": "@edx/frontend-logging",
   "version": "1.0.0-semantically-released",
-  "description": "Frontend logging client code for sending logging and events to New Relic",
-  "main": "dist/main.js",
+  "description": "Edx Frontend logging client code for sending logging and events to Datadog",
+  "main": "dist/index.js",
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
-    "build": "NODE_ENV=production BABEL_ENV=production webpack --config webpack.config.js",
-    "gc": "commit",
-    "commitmsg": "commitlint -e $GIT_PARAMS",
-    "coveralls": "cat ./coverage/lcov.info | coveralls",
-    "lint": "eslint --ext .js --ext .jsx .",
-    "is-es5": "es-check es5 ./dist/*.js",
-    "precommit": "npm run lint",
-    "prepublishOnly": "npm run build",
-    "semantic-release": "semantic-release",
-    "test": "jest --coverage",
-    "travis-deploy-once": "travis-deploy-once"
+    "build": "./node_modules/.bin/fedx-scripts babel src --out-dir dist --source-maps --copy-files",
```

Comment:
The `dist` directory contains more files than should be published. For example, the `setupTest.js` file nor any test files should be published, as it's not relevant to consumers' use of the package. The `dist` build output is also missing the `LICENSE`, `README`, and `package.json`.

<img width="221" alt="image" src="https://github.com/edx/frontend-logging/assets/2828721/0378b0be-0fb2-4c9d-8221-68090df3122f">

See the [previously linked `build` Makefile target](https://github.com/openedx/frontend-platform/blob/05e6820f627ac0d3e4dcf68858191f2a5b8e42c6/Makefile#L13-L21) in frontend-platform for reference on what the `build` NPM script would ideally be doing here.

```
"build": "make build",
```

Reply:
should we add makefile here as well?

Reply:
Yes. Given there's multiple commands to `rm` and `cp`, the `Makefile`s `build` target is preferable over having the copying/removal of files all in one line within `package.json`.